client/viame-web-common/components/Viewer.vue

```diff
@@ -287,14 +287,11 @@ export default defineComponent({
       if (pendingSaveCount.value > 0) {
         result = await prompt({
           title: 'Save Items',
-          text: 'There is unsaved data, what would you like to do?',
-          positiveButton: 'Save',
-          negativeButton: 'Discard',
+          text: 'There is unsaved data, would you like to continue or cancel and save?',
+          positiveButton: 'Continue',
+          negativeButton: 'Cancel',
```

Comment:
I still think cance/continue is ambiguous, and people probably won't read the text.

* Continue leaving, or continue editing?
* Cancel edits or cancel leaving?

How about something more explicit?

![Screenshot from 2020-11-13 16-23-21](https://user-images.githubusercontent.com/4214172/99122608-acf18e80-25cc-11eb-8135-83ac3e988fb5.png)